kore/src/Kore/Builtin/AssociativeCommutative.hs

```diff
@@ -0,0 +1,1450 @@
+{- |
+Module      : Kore.Builtin.AssociativeCommutative
+Description : Built-in
```

Comment:
Just to clarify, this module would not be suitable to handle general associative-commutative operations, just Map-like collections, is that right?